woocommerce/wc-admin
# Add timeline separator/heading to the sidebar

Closes #47.

This tiny PR adds a simple component for timeline separation (`Today`, `Yesterday`, `2 Days Ago`, `Last Week` headings). There are some basic styles for now which can develop as we start to build out everything.

I've also moved the sidebar code from `client/layout/sidebar.js` to `client/layout/sidebar/` since I expect us to have more things added here in the future. 

Also not 100% sure about naming it "Separator" since it acts as a heading for the cards under it. Maybe `TimelineHeading`?

To Test:
* Verify you see the example 'Today' heading on the sidebar. That's it for now, until we start building out the actual display.
